web/app/js/components/Sidebar.jsx

```diff
@@ -6,7 +6,7 @@ import './../../css/sidebar.css';
 
 export default class Sidebar extends React.Component {
   render() {
-    let normalizedPath = this.props.location.pathname.replace(/\/$/, "");
+    let normalizedPath = this.props.location.pathname.replace(/\/api\/v1\/namespaces\/.*\/proxy/g, "");
```

Comment:
Thanks for tracking this down. Does this break highlighting for local development (since the prefix does not match for, say, localhost:8084/servicemesh)? I think it would be better to use `this.props.pathPrefix` to do this replacement, since that should be set properly in all environments.

Reply:
I tested it locally and it worked, but I like the `pathPrefix` approach a lot better. Will change it, thanks!